proto/admin/v1/service.proto

```diff
@@ -0,0 +1,93 @@
+syntax = "proto3";
+package admin.v1;
+
+import "google/protobuf/empty.proto";
+
+option go_package = "github.com/ava-labs/avalanchego/proto/pb/admin/v1;adminv1";
+
+// AdminService is the API service for node admin management
+service AdminService {
+  rpc StartCPUProfiler(google.protobuf.Empty) returns (google.protobuf.Empty);
+  rpc StopCPUProfiler(google.protobuf.Empty) returns (google.protobuf.Empty);
+  rpc MemoryProfile(google.protobuf.Empty) returns (google.protobuf.Empty);
+  rpc LockProfile(google.protobuf.Empty) returns (google.protobuf.Empty);
+  rpc Alias(AliasArgs) returns (google.protobuf.Empty);
+  rpc AliasChain(AliasChainArgs) returns (google.protobuf.Empty);
+  rpc GetChainAliases(GetChainAliasesArgs) returns (GetChainAliasesReply);
```

Comment:
RPC naming convention is typically `*Request/*Response`. Can we use that naming convention in our proto types? You should be able to rename if you install the proto plugin in your ide + use the aforementioned refactor tool